pkg/cmd/cli/describe/deployments.go

```diff
@@ -93,19 +93,30 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string, settings kc
 			if err == nil {
 				sorted := deploymentsHistory.Items
 				sort.Sort(sort.Reverse(rcutils.OverlappingControllers(sorted)))
+				counter := 0
 				for _, item := range sorted {
 					if item.Name != deploymentName && deploymentConfig.Name == deployutil.DeploymentConfigNameFor(&item) {
 						header := fmt.Sprintf("Deployment #%d", deployutil.DeploymentVersionFor(&item))
 						printDeploymentRc(&item, d.kubeClient, out, header, false)
+						counter++
+					}
+					// Only show last three deployments here
+					if counter == 2 {
```

Comment:
Oh, I didn't realize we are on the same package. Yes, please, make it a constant.